cloud_run_service_custom_domain_mapping/main.tf

```diff
@@ -0,0 +1,31 @@
+# [START cloudrun_custom_domain_mapping_run_service]
+resource "google_cloud_run_service" "default" {
+  name     = "cloud-run-srv"
+  location = "us-central1"
+  template {
+    spec {
+      containers {
+        image = "gcr.io/cloudrun/hello"
+      }
+    }
+  }
+  traffic {
+    percent         = 100
+    latest_revision = true
+  }
+}
+# [END cloudrun_custom_domain_mapping_run_service]
+# [START cloudrun_custom_domain_mapping]
+data "google_project" "project" {}
+
+resource "google_cloud_run_domain_mapping" "default" {
+  name     = "verified-domain.com"
```

Comment:
We may need to skip the test for this. It's failing with `Error waiting to create DomainMapping: resource is in failed state "Ready:False", message: Caller is not authorized to administer the domain 'verified-domain.com'. If you own 'verified-domain.com', you can obtain authorization by verifying ownership of the domain, or any of its parent domains, via the Webmaster Central`

Reply:
This is expected.  I was able to test with an actual verified domain and it was able to complete.  If there are other resources that expect a verified domain, it might be worth implementing an exception in the tests.

Reply:
@rogerthatdev I meant we should add an exception like https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/cloud_run_service_add_tag/test.yaml since this will not pass in CI?